app/models/manageiq/providers/base_manager.rb

```diff
@@ -17,5 +19,11 @@ def ext_management_system
     def refresher
       self.class::Refresher
     end
+
+    private
+
+    def require_credentials
+      errors.add(:authentications, 'You must enter credentials.') if missing_credentials?
```

Comment:
I feel like validations that require child objects to exist can be potentially problematic.. but maybe this is okay?